### PR TITLE
Fixes #774. Add note about chrome apps and csp

### LINKED ISSUE #774: [vulcanize] please add
<https://www.polymer-project.org/articles/concatenating-web-components.html>

Please add advice for people writing packaged apps and extensions. I'm looking for something along the lines of :

<https://www.polymer-project.org/articles/concatenating-web-components.html#is-concatenation-a-good-idea>

I suppose the author might consider that to be enough, in which case never mind. It'd be nice to get some more guidance specific to our use-case, or at least a mention, so we know it applies to us too.

Most of the page assumes there are network downloads, but that's not always true.

### PR DESCRIPTION
**Pull request by robdodson**
@arthurevans PTAL

### CODE REVIEW

File: 1.0/tools/optimize-for-production.md [hunk 1]
```diff
@@ -200,7 +200,9 @@ This results in a `elements.vulcanized.html` that looks a little like this:
 
 [Content Security Policy](http://en.wikipedia.org/wiki/Content_Security_Policy) (CSP) is a JavaScript security model that aims to prevent XSS and other attacks. In so doing, it prohibits the use of inline scripts.
 
-To use {{site.project_title}} in a CSP environment that doesn't support inline scripts, you can use the Crisper project. Crisper removes all scripts from the HTML Imports and places their contents into an output JavaScript file. This is useful in amongst other things, using {{site.project_title}} in a Chrome App.
+The Polymer build tools can also be used to transform your code to run in an environment that enforces content security policy (CSP), including Chrome Apps and Extensions.
```
Comment (arthurevans) [inline 1.0/tools/optimize-for-production.md]: Can we move this paragraph right up to the top (like, L14), so the intro to this doc explicitly mentions CSP (maybe with a link to this section)?

File: 1.0/tools/optimize-for-production.md [hunk 2]
```diff
@@ -200,7 +200,9 @@ This results in a `elements.vulcanized.html` that looks a little like this:
...
+
+To use {{site.project_title}} in a CSP environment, you can use the Crisper project. Crisper removes all scripts from the HTML Imports and places their contents into an output JavaScript file. This is useful in amongst other things, using {{site.project_title}} in a Chrome App.
```
Comment (arthurevans) [inline 1.0/tools/optimize-for-production.md]: For clarity: maybe s/output JavaScript file/external JavaScript file/?

For second sentence, maybe: "This lets you use Polymer in any environment that enforces CSP, including Chrome Apps and Extensions."

**Comment by arthurevans** [comment]
Suggested a couple of tweaks to make this more prominent.

**Comment by robdodson** [comment]
Rebase @arthurevans PTAL

**Comment by arthurevans** [comment]
:heart: :it: :dog:

**Pull request merged**